backend/followers/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+from authors.models import Author
+
+class Follower(models.Model):
+    follower = models.ForeignKey(Author, on_delete=models.CASCADE, related_name='follower')
```

Comment:
@jonny21099 Should followers be structured like this? After defining this, `author.follower_set.all()` couldn't be found. How can we get the `follower_set` working?

Reply:
The reason why it doesn't work is because you defined a related_name='follower' so the code should be author.follower.all(), that should work, when I tried it on my end

Reply:
Fixed